namui/src/namui/common/mod.rs

```diff
@@ -173,3 +174,15 @@ pub struct TypefaceType {
     pub language: Language,
     pub font_weight: FontWeight,
 }
+
+#[derive(Debug, Hash, Eq, PartialEq, Clone, Copy)]
+pub enum MouseButton {
+    Left,
+    Middle,
+    Right,
+}
+
+pub struct RawMouseEvent {
+    pub xy: Xy<f32>,
+    pub buttons: HashSet<MouseButton>,
```

Comment:
If HashSet is too heavy, I can make light struct for it using bit-wise operator. But I don't think that we face the performance issue right now.